web/src/components/data-fields/address/AddressMunicipalityDropdown.test.tsx

```diff
@@ -10,12 +10,12 @@ describe("<AddressMunicipalityDropdown  />", () => {
     addressData,
     municipalities,
   }: {
-    addressData?: Address;
+    addressData?: FormationAddress;
```

Comment:
[sand] If we're changing this type, it might make sense to update the variable name to match.
```suggestion
    formationAddressData?: FormationAddress;
```